ci/unit/test_post_always_node.sh

```diff
@@ -5,10 +5,10 @@ set -uex
 cd "$DAOS_BASE"
 mkdir run_test.sh
 mkdir vm_test
-mv nlt-errors.json vm_test/
 if ls /tmp/daos*.log > /dev/null; then
   mv /tmp/daos*.log run_test.sh/
 fi
+mv nlt-errors.json vm_test/
```

Comment:
This makes a lot of sense, if it's going to fail at this point then we do want the logs run run_test.sh.  The "dnt" logs are created by the same code as this file so are unlikely to exist if this file doesn't.

Reply:
This is better because we now don't loose the logs but unfortunately the step to publish them in Test Results is not reached because CI is aborted by the same "mv" command.

Reply:
Ah, so if run_test.sh (the script) fails we get the test results in Jenkins, but not the logs?  Is that what's happening?

I have to admit I tried looking for the logs from the unit test a few weeks ago and concluded that it wasn't saving any so perhaps this is what was going on then.

Reply:
What happens right now in master is every time the test_run.sh script executes the tests, each test saves it´s log in /tmp/ under the name "daos_*.log", once it finishes, if there were failures, the test_main_node.sh script (which invoked the run_test.sh script) exits immediately without running the NLT tests.
Then, the test_post_always_node.sh script aborts CI when trying to move the nlt-errors.json file to vm_test/ because it wasn't created, so the next step which moves all "daos_*.log" files to "run_tes.sh/" directory never executes, but the logs remain in /tmp.

With respect to Jenkins, we cannot see the results there if runt_test.sh stage aborts CI because the step that publish those results is never reached (I believe is the stepResult.groovy, I haven't confirmed this yet).